lib/schmooze/version.rb

```diff
@@ -1,3 +1,3 @@
 module Schmooze
-  VERSION = "0.1.6"
+  VERSION = "0.1.7"
```

Comment:
Sorry one more change, the VERSION should be `0.2.0` because this adds a new feature

Reply:
No problem, You are right.